Tools/DumpRenderTree/TestOptions.cpp

```diff
@@ -93,6 +93,7 @@ const TestFeatures& TestOptions::defaults()
             { "RequiresUserGestureForAudioPlayback", false },
             { "RequiresUserGestureForMediaPlayback", false },
             { "RequiresUserGestureForVideoPlayback", false },
+            { "ScrollToTextFragmentIndicatorEnabled", false },
```

Comment:
What about WKTR?

Reply:
@hortont424 do you know where I would disable it? I still haven't figured out where in WKTR to do so